node/api.go

```diff
@@ -78,6 +78,24 @@ func (api *privateAdminAPI) AddPeer(url string) (bool, error) {
 	return true, nil
 }
 
+type MaxConcReq struct {
+	HttpLimit uint64 // max concurrent requests for http requests. 0(Default) means no limit
+	WSLimit   uint64 // max concurrent requests for ws requests. 0(Default) means no limit
+}
+
+// get max concurrent requests parameter for rpc requests
+func (api *privateAdminAPI) GetMaxConcReq() *MaxConcReq {
+	httpLimit := api.node.http.httpConfig.maxConcReq
```

Comment:
For changing these values online via RPC, I believe, we'd need to have them atomic. How do you thing?

Reply:
Done ✅